lada/fellow/forms.py

```diff
@@ -57,6 +61,8 @@ class EditForm(FlaskForm):
     fotki = BooleanField('Zdjęcia')
     fszysko = BooleanField('Fszysko')
 
+    rodo = BooleanField(rodo_cyrograf)
```

Comment:
Nie trzymasz informacji, jaką klauzulę RODO się podpisało. W przypadku przypału nie będzie możliwości wykazać, którą wersję człowiek podpisał.

Reply:
hmm, to się robi wtedy trudniejsze, masz jakiś pomysł jak to zrobić rozsądnie?